.github/actions/fastlane/action.yaml

```diff
@@ -0,0 +1,82 @@
+name: 'fastlane'
+description: 'Runs fastlane with target'
+inputs:
+  environment:
+    description: "According .env file provided in fastlane folder, usually one of [local, dev, staging, prod]"
+    required: true
+  platform:
+    description: "Build platform declared in Fastfile, usually one of [android, ios]"
+    reauired: true
+  target:
+    description: "Build target declared in Fastfile, usually one of [build, release]"
+    required: true
+  # Optional
+  ruby_version:
+    required: false
+    default: '3.0'
+  bundler_version:
+    description: Option for 'gem install` command
+    required: false
+    default: '2.2.16'
+  # Android
```

Comment:
separate this file into 2
one for ios
one for android 

so we don't have a single pipeline that covers all. That means "platform" argument can be gone.

Reply:
Done